site/docs/operations/cluster-sizing.md

```diff
@@ -191,6 +191,13 @@ Memory consumption is affected by:
   data. See the [Hazelcast IMDG Deployment and Operations Guide](https://hazelcast.com/resources/hazelcast-deployment-operations-guide/)
   .
 
+> Hazelcast IMDG offers lite members to prevents the memory usage on
```

Comment:
"to prevent memory usage"

Reply:
I think lite members were mainly introduced as a more performant client or to server as executors. Your cluster won't use any less memory if you add a lite member to it.

https://docs.hazelcast.org/docs/4.0.3/manual/html-single/index.html#enabling-lite-members

I think they are just ignored by Jet.

Reply:
I did not see any benefit from using these lite members. I thought the user must have an extra reason to use a nonhomogenous cluster of lite members and data members, e.g. a user may want to avoid memory usage on some of the cluster members.